app/assets/javascripts/calendars/appointment-availability.es6

```diff
@@ -59,19 +59,46 @@
 
       super(el, calendarConfig);
 
+      this.coloursConfig = [
+        { count: 30, colour: 'hsl(200, 50%, 25%)' },
+        { count: 20, colour: 'hsl(200, 50%, 35%)' },
+        { count: 10, colour: 'hsl(200, 50%, 45%)' },
+        { count: 5,  colour: 'hsl(200, 50%, 55%)' },
+        { count: 1,  colour: 'hsl(200, 50%, 65%)' }
+      ];
+
       this.init();
       this.addEvents();
     }
 
+    getEventColour(event) {
+      if (event.selected === true) {
+        return this.selectedEventColour;
+      }
+
+      const colours = $.grep(this.coloursConfig, (configItem) => {
```

Comment:
Yeah I don't think we need to worry too much about the resolution of the colour bands. If there's a slight precedence to book one with more availability then great, but the customer is still able to book any slot that's available regardless.